src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -47,10 +47,17 @@ public CommandResult execute() {
      * @return list of persons found
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
+        Set<String> keywordsInLowerCase = new HashSet<>();
+        for (String keyword : keywords) {
+            keywordsInLowerCase.add(keyword.toLowerCase());
```

Comment:
Do you think that this could be improved upon? Perhaps while the parser is adding the set of keywords, change the words to lowercase?